Baidu Scholar.js

```diff
@@ -106,16 +110,63 @@ function scrape(doc, url) {
 				item.abstractNote = text(doc, 'div.sc_abstract') || text(doc, 'p.abstract');
 			}
 			item.attachments.push({
-				title: "Snapshot",
-				document: doc
+				url,
+				title: "Baidu学术地址",
+				mimeType: "text/html",
+				snapshot: false,
 			});
+			// Sources
+			var sources = []
+			doc.querySelectorAll('div#allversion_wr span a:not(.paper_more_btn)').forEach(e => {
+				sources.push({
+					title: ZU.trimInternal(e.textContent),
+					url: e.href
+				})
+			})
+			var related = []
+			doc.querySelectorAll('div#related_lists a').forEach(e => {
+				related.push({
+					title: ZU.trimInternal(e.textContent),
+					url: e.href
+				})
+			})
+			// Z.debug({
+			// 	sources, related
+			// })
+			item.tags = tags
+			if (item.title == null) {
+				item.title = title
+			}
+			if (!item.creators || item.creators.length == 0) {
+				item.creators = []
+				doc.querySelectorAll('p.author_text a').forEach(e => {
+					var name = ZU.trimInternal(e.textContent)
+					var index = name.indexOf(',')
+					var lastName = name.slice(0, index)
+					var firstName = name.slice(index+1)
+					item.creators.push({ lastName, firstName, creatorType: 'author' })
+				})
+			}
+			if (!item.publicationTitle && attr(doc, 'a.journal_title', 'title')) {
+				item.publicationTitle = attr(doc, 'a.journal_title', 'title')
+			}
+			if (!item.date && text(doc, 'div.year_wr p.kw_main')) {
+				item.date = ZU.trimInternal(text(doc, 'div.year_wr p.kw_main'))
+			}
+			if (item.DOI == null && text(doc, 'div.doi_wr p.kw_main')) {
+				item.DOI = ZU.trimInternal(text(doc, 'div.doi_wr p.kw_main'))
+			}
+			if (!item.extra && text(doc, 'p.ref-wr-num a')) {
+				let n = ZU.trimInternal(text(doc, 'p.ref-wr-num a'))
+				item.extra = "ZSCC:"+Array(7-n.length).fill('0').join('')+n
```

Comment:
What is ZSCC?

Reply:
It is from [zotero-scholar-citations](https://github.com/MaxKuehn/zotero-scholar-citations). Thought it was standard. Should I leave it empty?

Reply:
Yes, let's remove this please and update tests accordingly